api/src/db/migrations/v107_refactor_interstate_transport_essential_question.ts

```diff
@@ -0,0 +1,414 @@
+import { v106UserData } from "./v106_add_pet_care_housing_essential_question";
+
+export interface v107UserData {
+  user: v107BusinessUser;
+  profileData: v107ProfileData;
+  formProgress: v107FormProgress;
+  taskProgress: Record<string, v107TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v107LicenseData | undefined;
+  preferences: v107Preferences;
+  taxFilingData: v107TaxFilingData;
+  formationData: v107FormationData;
+  lastUpdatedISO: string | undefined;
+  version: number;
+}
+
+export const migrate_v106_to_v107 = (v106Data: v106UserData): v107UserData => {
+  const getLogisticsResponse = () => {
+    if (
+      v106Data.profileData.industryId === "logistics" &&
+      v106Data.profileData.interstateTransport === true
```

Comment:
Brought this up in techtro last week, we don't want to be checking `=== true` or `=== false` and instead we should simplify the expression